### PR TITLE
docs: add doc comments for methods used to create a Directory in Alluxio

### PR DESCRIPTION
**Pull request by abnerferreira0212**
# Pull Request Merge Checklist
           
Task backlog identifier related to this PR: [Issue #82](https://github.com/s1mbi0se/simbiose_knowledge_base/issues/82)

---

## As a DEVELOPER, I confirm that this PR **doesn't contain**:
- [x] Code lines with syntax errors
- [x] Methods and variables names that aren't clear or doesn't mean anything
- [x] Code that is out of the standard (Example: **PEP-8** for Python code)
- [x] Files with **imports** errors or without any utility
- [x] **TODOs** left behind
- [x] Tests that are being skipped
- [x] Methods placed on incorrect files or folders (Example: methods that are local, but should be placed on /utils)
- [x] Functions, methods or API responses that are publicly returning sensitive information

## As a DEVELOPER, I confirm that I:
- [x] Followed the Simbiose's standard for branch and commit naming
- [x] Concluded the task and achieved the goals according to the task planning I defined before starting to code
- [x] Added all necessary database changes to the [SQL Migrations](https://github.com/s1mbi0se/sql-migrations)
- [x] Created tests for every new flow and method on this Pull Request
- [x] Added the necessary DEBUG logs
- [x] Checked my code using the Simbiose's Corner Cases Guide
- [x] Merged the **develop** branch to this PR/branch before sending the PR for review
- [x] Solved all possible merge conflicts before sending the PR for review
- [x] Corrected all the problems and code smells identified by the quality tools
- [x] Manually executed the necessary tests locally to make sure my changes didn't break anything
- [x] All unitary and integration tests ran by the CI system are passing
- [x] Check the performance of my changes or additions and they didn't make the system slower (I profiled the code)
- [x] Updated the necessary documentation (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Made the self-review of the code before sending the PR for review
- [x] Sent this PR for a peer-review
- [x] Made all the corrections appointed during the peer-review
- [x] Confirmed that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block my merge

---  

## As the TEAM MANAGER, I confirm that I:
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked if the purpose of the task was reached with this PR
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if all related PRs were created when other components were affected by this task (e.g. database changes on [SQL Migrations](https://github.com/s1mbi0se/sql-migrations))
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s pattern for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

---  

## As the QUALITY RESPONSIBLE, I confirm that I:
- [ ] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [ ] Checked if there is no possible merge conflicts before reviewing the PR
- [ ] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [ ] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [ ] Checked that code coverage for this PR is higher than 80%.
- [ ] Checked that there are no problems and code smells identified by the quality and security tools
- [ ] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [ ] Checked that all classes, methods and variables on this PR follow the Simbiose’s code standard for this programming language
- [ ] Checked that all unitary and integration tests ran by the CI system are passing
- [ ] Checked that there are no performance issues created by these changes or additions
- [ ] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [ ] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [ ] Checked all GitHub Actions related to this PR that can block the merge

## Important links:
1. [Simbiose Quality Guidelines](https://github.com/s1mbi0se/company/wiki/Quality-Topics#table-of-contents)
2. [Metadata Database Modeling](https://docs.google.com/spreadsheets/d/1Y8_k5xV86kTjgfjbi-PGsJcubUKhWlT2x8ecpnMlyR0/edit?usp=sharing)
3. [Simbiose's Corner Cases Guide](https://www.simbioseventures.com/documentos-e-guias/procedimentos-e-checklists/corner-cases-guide)

### CODE REVIEW

File: core/common/src/main/java/alluxio/client/file/URIStatus.java [hunk 1]
```diff
@@ -131,7 +133,9 @@ public int getInAlluxioPercentage() {
   }
 
   /**
-   * @return the epoch time the entity referenced by this uri was last modified, mutable
+   * Gets the time the entity referenced by this URI was last modified.
+   *
+   * @return  the epoch time the entity referenced by this uri was last modified, mutable
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/client/file/URIStatus.java:138]: Keep consistency with lower/upper case
```suggestion
   * @return  the epoch time the entity referenced by this URI was last modified, mutable
```

File: core/common/src/main/java/alluxio/wire/FileInfo.java [hunk 2]
```diff
@@ -157,7 +157,15 @@ public boolean isFolder() {
   }
 
   /**
-   * @return whether the file is pinned
+   * Checks whether the file is pinned.
+   * <p>
+   * Returns a boolean representing whether the
+   * file is {@link #mPinned}. Returning {@code true}
+   * means the file is pinned, and {@code false} means
+   * it is not
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/wire/FileInfo.java:165]: missing final point.

File: core/server/common/src/main/java/alluxio/master/journalv0/JournalEntryRepresentable.java [hunk 3]
```diff
@@ -18,7 +18,13 @@
...
 public interface JournalEntryRepresentable {
   /**
-   * @return the {@link JournalEntry} representing this object
+   * Returns the journal entry representation of this object.
+   * <p>
+   * Implementations should return an object of type {@link JournalEntry}
+   * to represent the instance of the class that implements this
+   * {@link JournalEntryRepresentable} interface.
+   *
+   * @return  the {@link JournalEntry} representing this object
```
Comment (caiopavanelli) [inline core/server/common/src/main/java/alluxio/master/journalv0/JournalEntryRepresentable.java:27]: Don't repeat links in javadoc. Do for the first occurrence only.

File: core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java [hunk 4]
```diff
@@ -377,13 +377,15 @@ public LockedInodePath lockInodePath(LockingScheme scheme)
...
+   * Locks all {@link Inode}s on the provided {@code uri} using the specified {@code lockPattern}.
+   * <p>
+   * The target inode is not required to exist.
    *
-   * @param uri the uri to lock
-   * @param lockPattern the {@link LockPattern} to lock the inodes with
-   * @return the {@link LockedInodePath} representing the locked path of inodes
-   * @throws InvalidPathException if the path is invalid
+   * @param   uri         the {@link AlluxioURI} to lock
+   * @param   lockPattern the {@link LockPattern} to lock the inodes with
```
Comment (caiopavanelli) [inline core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java:387]: Don't repeat {@link LockPattern}

File: core/server/master/src/main/java/alluxio/master/file/meta/InodeView.java [hunk 5]
```diff
@@ -57,7 +57,12 @@
   long getTtl();
 
   /**
-   * @return the {@link TtlAction}
+   * Gets the time-to-live (TTL) action.
+   * <p>
+   * Implementations should return the {@link TtlAction} for
+   * this {@link InodeView}.
+   *
+   * @return  the {@link TtlAction} for this {@link InodeView}
```
Comment (caiopavanelli) [inline core/server/master/src/main/java/alluxio/master/file/meta/InodeView.java:65]: Don't repeat links.

File: core/server/master/src/main/java/alluxio/master/file/meta/LockedInodePath.java [hunk 6]
```diff
@@ -205,6 +208,11 @@ public Inode getLastExistingInode() {
   }
 
   /**
+   * Gets a copy of the list of existing inodes from the root.
+   * <p>
+   * Returns a copy of the existing {@link #mLockList} with the
+   * existing {@link InodeView}s from the root {@link Inode}.
```
Comment (caiopavanelli) [inline core/server/master/src/main/java/alluxio/master/file/meta/LockedInodePath.java:214]: Don't repeat links.

File: core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java [hunk 7]
```diff
@@ -249,10 +249,12 @@ public String getMountPoint(AlluxioURI uri) throws InvalidPathException {
   }
 
   /**
-   * Returns a copy of the current mount table, the mount table is a map from Alluxio file system
+   * Gets a copy of the current mount table.
+   * <p>
+   * Returns a copy of the current mount table, which is a map from Alluxio file system
```
Comment (caiopavanelli) [inline core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java:254]: Missing final point.

File: core/server/master/src/main/java/alluxio/master/metastore/InodeStore.java [hunk 8]
```diff
@@ -144,13 +151,18 @@ default boolean supportsBatchWrite() {
...
-   *
-   * This method requires an inode lock manager read or write locks on the added edge.
-   *
-   * @param parentId the parent id
-   * @param childName the child name
-   * @param childId the child inode id
+   * Makes an inode the child of another inode.
+   * <p>
+   * Implementations should make the inode with the provided
+   * {@code childId} and {@code childName} the child of the inode
```
Comment (caiopavanelli) [inline core/server/master/src/main/java/alluxio/master/metastore/InodeStore.java:157]:
```suggestion
   * {@code childId}, {@code childName} and the child of the inode
```

File: core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java (hunk 7 above)
Comment (abnerferreira0212) [reply core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java:254]: The sentence ends in the following line.
